.gitignore

```diff
@@ -86,7 +86,14 @@ npm-debug.log.*
 .vscode/
 *.swo
 
-submodules/tests
+submodules/tests
 scripts/reporting/statsGen/yarn.lock
 scripts/reporting/statsGen/*/*.csv
 scripts/reporting/statsGen/*/*/*.csv
+scripts/reporting/statsGenElastic/yarn.lock
```

Comment:
in general for 3drepo.io, we removed yarn.lock because there was some libraries that we use which would always conflict with different machine types (what worked on a windows machine produced an incompatible yarn.lock file for mac)

So basically everytime we merged in code, depending on who committed it you may have to remove the yarn.lock file because it's incompatible. So we removed it in favour of locking down every version in package.json